Jasmine/Common/Helpers/SoundService.swift

```diff
@@ -8,8 +8,21 @@ class SoundService: NSObject {
 
     static let sharedInstance = SoundService()
 
-    var backgroundVolume: Float = Volume.max
-    var effectVolume: Float = Volume.max
+    var backgroundVolume: Float = Volume.max {
+        didSet {
+            backgroundPlayers.forEach { _, player in
```

Comment:
Can change it to `for (_, player) in backgroundPlayers` to make it easier to read? Same as below.